facade/interface.go

```diff
@@ -69,6 +69,7 @@ type NodeHandler interface {
 // ApiResolver defines a structure capable of resolving REST API requests
 type ApiResolver interface {
 	ExecuteSCQuery(query *process.SCQuery) (*vmcommon.VMOutput, error)
+	ExecuteQueryWithCallerAndValue(query *process.SCQuery, callerAddr []byte, callValue *big.Int) (*vmcommon.VMOutput, error)
```

Comment:
change the SCQuery data - add callerAddr and callValue inside - no need to change the interface and add these values

Reply:
done